presto-native-execution/scripts/release-centos-dockerfile/README.md

```diff
@@ -0,0 +1,492 @@
+# Prestissimo - Dockerfile build
+
+> 💡 _**PrestoDB** repository: [Presto - https://github.com/prestodb/presto](https://github.com/prestodb/presto)_
+
+> 💡 _**Velox** repository: [Velox - https://github.com/facebookincubator/velox](https://github.com/facebookincubator/velox)_
+
+## Practical Velox implementation using PrestoCpp
+
+> 📝 _**Note:** This readme and the build process was adapted from internal pipeline. You can e-mail the author if you've got questions [milosz.linkiewicz@intel.com](mailto:milosz.linkiewicz@intel.com)_
+
+Prestissimo, marked in PrestoDB GitHub repository as 'presto-native-execution', is effort of making PrestoDB even better using Velox library as a starting point. Both of mentioned - PrestoCpp and Velox - are mainly written using low level `C` and `C++ 17` languages, which makes the build-from-scratch process humongously complicated. To make this process simple, Intel Cloud Native Data Services Team is introducing 3-stage, fully automated Docker build process based on unmodified project GitHub repository.
```

Comment:
> which makes the build-from-scratch process humongously complicated

We are working hard to simplify this though :)

Reply:
I know :-p, but most of the folks don't know C++ at all so this addresses them.